torch/csrc/jit/passes/graph_fuser.cpp

```diff
@@ -209,7 +209,12 @@ struct GraphFuser {
       return false;
     if (!node->is_constant(attr::dim))
       return false;
+
     auto tensors_node = node->namedInput(attr::tensors)->node();
+    if( (tensors_node->inputs().size() + node->outputs().size()) >
```

Comment:
FWIW there's always only a single output in here, so we don't have to account for that

Reply:
Since an output could still be an argument to the fused kernel generated later, I didn't change this part in the latest commit.

Reply:
BTW we should assert that this never happens, because now we have guards for it. If it does, then someone messed up somewhere.

Reply:
Would you suggest a specific place and assert to add?

Reply:
Right here. This is the condition that should never happen in this very place